### ISSUE TITLE
Proposal how to cleanup and manage open PRs

### ISSUE THREAD

**Issue by certik** [sympy/sympy#24110]
In LFortran I started marking all PRs that need PR author's response (whether discussion or code) as Draft, and when ready, the PR author clicks the "Ready for review" button. So PRs that are waiting for review are:

https://github.com/lfortran/lfortran/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse

And all other PRs that are waiting for author's response are:

https://github.com/lfortran/lfortran/pulls?q=is%3Apr+is%3Aopen+draft%3Atrue

This seems like a very simple system that solves the problem of having too many PRs and not sure which one the repository maintainers should focus on.

**Comment by JASKIRAT11011** [comment]
@certik plzz assign me the following issue

**Comment by certik** [comment]
Indeed, I created those "author's turn" and "sympy's turn" issues to solve this problem, but exactly as you say, they become out of date and the PR authors cannot change them, so it does not fix the problem. Draft/Non-Draft status however can be changed by the PR author, so I think this will work.

Regarding SymPy too broad, one just needs more maintainers, perhaps in a specific subset of SymPy.

**Comment by oscarbenjamin** [comment]
So what do you suggest as a workflow then?

When reviewing a PR mark it as draft if it is not ready to be merged?

**Comment by certik** [comment]
I've tested this workflow in the past 3 weeks and it works great. I simply add `draft:false` into the PR search bar, and only see the "green" PRs. Of those, I click one by one, review, if more work is needed, mark it as draft, if it is ready to be merged, then I merge it. For some that are almost ready, I leave them there. Then next day I do the same, if the PR is still there, I simply mark it as Draft. That way I clean up my plate completely. I caught up with some old PRs that I didn't have time to review. So I think this is the first workflow that actually works for me.
